README.md

```diff
@@ -1,2 +1,37 @@
 # lightbeam-we
-Web Extension version of the Mozilla Lightbeam add-on
+This is the web extension version of the Mozilla Lightbeam add-on for visualizing HTTP requests between websites in real time.
+
+The Mozilla Lightbeam extension is a key tool for Mozilla to educate the public about privacy
+
+## Quick Start
+
+* Clone the repository
```

Comment:
I would say a more typical workflow might be to fork the repo, clone the fork (becomes `origin`) and sync the repo as `upstream` in Git? This is the workflow @jkt advised for me for my development work on Lightbeam anyway! It means less notifications/noise if people are experimenting and making DNM commits, I believe.